frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1010,6 +1010,7 @@ function ReaderRolling:updatePos()
         logger.dbg("rendering hash changed:", self.rendering_hash, ">", new_rendering_hash)
         self.rendering_hash = new_rendering_hash
         -- A few things like page numbers may have changed
+        self.ui.document:resetCallCache() -- be really sure this cache is reset
```

Comment:
ReaderRolling is CRe only, right? No chance this would blow up elsewhere ;).

Reply:
Welcome to the project ! (too much low level Kobo Elipsa I assume :)
The anwser is yes: rolling is cre only :) (self.ui.document:getDocumentRenderingHash() above this would have blown up before :)